buildenv/jenkins/docker-slaves/jenkinsfile-build-container.groovy

```diff
@@ -63,17 +63,24 @@ timeout(time: 5, unit: 'HOURS') {
                 }
                 stage("Build") {
                     if(params.ghprbPullId) {
-                        TAGS = "-t ${REPOSITORY}:PR${ghprbPullId}"
+                        TAGS = "--tag=${REPOSITORY}:PR${ghprbPullId}"
                     } else {
-                        TAGS = "-t ${REPOSITORY}:${BUILD_NUMBER} -t ${REPOSITORY}:latest"
+                        /* FIXME mkdocker.sh doesn't handle multiple --tag options */
```

Comment:
There should be an Issue for this.

Reply:
The script already supports multiple `--tag` options: the comment was just stale and has been removed.